src/bluesky/bundlers.py

```diff
@@ -426,6 +426,11 @@ def emit_event(readings: Optional[Dict[str, Reading]] = None, *args, **kwargs):
                     f"{readable_obj} has async read() method and the callback "
                     "passed to subscribe() was not called with Dict[str, Reading]"
                 )
+            now = ttime.time()
+            acceptable_delay = 0.1
```

Comment:
This needs to be configurable.

Passed in as an arg to the `Msg` when we create the monitor

Reply:
very sensible, will add soon

Reply:
added